linux_os/guide/services/cron_and_at/file_groupowner_cron_daily/policy/stig/shared.yml

```diff
@@ -0,0 +1,28 @@
+srg_requirement: |-
+    {{{ full_name }}} cron configuration files directory must be group-owned by root.
+
+vuldiscussion: |-
+    Service configuration files enable or disable features of their respective services that if configured incorrectly
+    can lead to insecure and vulnerable configurations. Therefore, service configuration files should be owned by the
+    correct group to prevent unauthorized changes.
```

Comment:
For some reason the VulDiscussion column is empty in the generated HTML. Can you investigate this problem?

Reply:
This should be fixed now.